apps/api/v2/src/modules/organizations/controllers/organizations.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { API_VERSIONS_VALUES } from "@/lib/api-versions";
+import { Roles } from "@/modules/auth/decorators/roles/roles.decorator";
+import { NextAuthGuard } from "@/modules/auth/guards/next-auth/next-auth.guard";
+import { OrganizationRolesGuard } from "@/modules/auth/guards/organization-roles/organization-roles.guard";
+import { OrganizationIdGuard } from "@/modules/auth/guards/organization/organization-id.guard";
+import { GetUsersInput } from "@/modules/organizations/controllers/inputs/get-users-input.dto";
+import { ListUsersResponseDto } from "@/modules/organizations/controllers/outputs/listUsersResponse.dto";
+import { OrganizationUsersRepository } from "@/modules/organizations/repositories/organizationUsers.repository";
+import { Controller, Get, Post, Param, Logger, UseGuards, Body } from "@nestjs/common";
+import { ApiExcludeController } from "@nestjs/swagger";
+
+import { ApiResponse } from "@calcom/platform-types";
+
+@Controller({
+  path: "/organizations",
+  version: API_VERSIONS_VALUES,
+})
+@ApiExcludeController(false)
```

Comment:
`@ApiExcludeController` can be removed - it has been used only in endpoints that should be excluded from the swagger docs. If we include endpoints, then we don't need `@ApiExcludeController(false)` as they will be included by default.